san/sanbase_graphql.py

```diff
@@ -1,9 +1,16 @@
 import iso8601
 import datetime
 
-DEFAULT_TO_DATE = lambda: datetime.datetime.now()
-DEFAULT_FROM_DATE = lambda: datetime.datetime.now() - datetime.timedelta(days=365)
+
+def DEFAULT_TO_DATE(): return datetime.datetime.now()
```

Comment:
Why do you remove the lambda ? It is there so the date is calculated when it is used, not when the library is loaded.

Reply:
I see but why is the change then ?

Reply:
Haha that was a linter change, to be honest IMO let's change the casing and keep it as a function but if you prefer I can change it back to a lambda.